book/src/language-features/README.md

```diff
@@ -0,0 +1,31 @@
+# Language Features
+
+Much SQL syntax is complicated. PRQL has facilities to simplify the statements.
+
+[Coalesce](./coalesce.md) - How PRQL handles potentially NULL values
```

Comment:
I think this page is excellent with the links! This is somewhat what I meant above.

Should we do either all the "index pages" using bullets or none of them?